### ISSUE TITLE
Vagrant Up Failures on Windows with VirtualBox

### ISSUE THREAD

**Issue by JasonWeakley** [geerlingguy/drupal-vm#1983]
### Issue Type

 - Bug Report / Support Request

### Your Environment

```
Vagrant 2.2.5

echo "VirtualBox `vboxmanage --version`"
>> ^C
(the "echo VirtualBox" command does not work on Windows)

ansible : The term 'ansible' is not recognized as the name of a cmdlet, function, script file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct and try again.
At line:1 char:1
+ ansible --version
+ ~~~~~~~
    + CategoryInfo          : ObjectNotFound: (ansible:String) [], CommandNotFoundException
    + FullyQualifiedErrorId : CommandNotFoundException
```

### Your OS

 - **Windows** 10

### Full console output

https://gist.github.com/JasonWeakley/3f085cb107596459ff7cd0e9e14681cf

### Summary

When I download the .zip file I open the default.config.yml and change the IP address and hostname and machine name:
```
vagrant_hostname: mdo2.dev
vagrant_machine_name: mdodrupal
vagrant_ip: 192.168.88.93
```
Then I edit the Vagrantfile and change line 117 to the below:
```
# Use pip to get the latest Ansible version when using ansible_local.
    provisioner == :ansible_local && ansible.install_mode = 'default'
```
Changed to 'default'
After that I run `vagrant up` and wait 10-15 minutes to see if the install fails or not, which it does 9 out of 10 times. Essentially, all I know is that Ansible failed, and VBGuests failed, but doing `vagrant reload` and `vagrant provision` solves nothing. As you can see at the bottom of the Gist link above I ran vagrant reload and got a false success message. What I see on my screen is a "Secure Connection Failed" page from Firefox, trying to load the host address as if it were a real website.

**Comment by JasonWeakley** [comment]
My steps and troubleshooting:
`vagrant plugin install vagrant-vbguest`
`vagrant up`
...wait
```
PLAY RECAP *********************************************************************
drupalvm                   : ok=187  changed=5    unreachable=0    failed=1    skipped=347  rescued=0    ignored=1

Ansible failed to complete successfully. Any error output should be
visible above. Please fix these errors and try again.
```
`vagrant reload`
```
==> drupalvm: Machine 'drupalvm' has a post `vagrant up` message. This is a message
==> drupalvm: from the creator of the Vagrantfile, and not from Vagrant itself:
==> drupalvm:
==> drupalvm: Your Drupal VM Vagrant box is ready to use!
==> drupalvm: * Visit the dashboard for an overview of your site: http://dashboard.mdothree.dev (or http://192.168.88.94)
==> drupalvm: * You can SSH into your machine with `vagrant ssh`.
==> drupalvm: * Find out more in the Drupal VM documentation at http://docs.drupalvm.com
```
`vagrant ssh`
`vagrant@mdothree:/var/www/drupalvm$ composer update drupal/core`
```
vagrant@mdothree:/var/www/drupalvm$ composer update drupal/core
    1/2:        http://repo.packagist.org/p/provider-latest$ecc5b6adda723bfb1c61851a3743761703e9576df2092147eafa76906932cb53.json
    Finished: success: 1, skipped: 0, failure: 1, total: 2
Package "drupal/core" listed for update is not installed. Ignoring.
Loading composer repositories with package information
Updating dependencies (including require-dev)
Nothing to install or update
Generating autoload files
```

**Comment by JasonWeakley** [comment]
I've also found this issue to be relevant:
https://github.com/geerlingguy/drupal-vm/issues/1970#issuecomment-537078135
I have added myself to the list, updated group policies with `gpupdate /force` and ran `vagrant provision` again.

**Comment by JasonWeakley** [comment]
@geerlingguy My simple understanding is that Ubuntu is used to install Virtual Box Guest Additions (correct?), and is it likely that it fails because some Ubuntu packages were not found at the following URLs?
https://gist.github.com/JasonWeakley/3f085cb107596459ff7cd0e9e14681cf#file-vagrant-up-failures-txt-L69
Lines 69-71 show failed fetching of `.deb` files. Now, I just tried to install another new box and ran `vagrant plugin install vagrant-vbguest` prior to running `vagrant up`. So, even with VB Guest Additions installed (manually) it fails to recognize that its there.

**Comment by JasonWeakley** [comment]
Here is more from today as I attempted to create a VM and failed, this output/gist shows repeated attempts to provision without success... and no idea how to proceed to solve the problem...
https://gist.github.com/JasonWeakley/23c9103adfb82af2c71b69c0b31402c6

**Comment by JasonWeakley** [comment]
OK, so I have concluded my scientific experiments and have come to this conclusion: "If at first you don't succeed, try, try again." I attempted yesterday to provision a box about half a dozen times, each time I came up against a problem where an Ubuntu package was not being installed, and I believe preventing PHP from being installed -- regardless PHP failed to install and my "PLAY RECAP" was showing something like ~60 "OK" and ~60 "skipped" and spinning up the box fails. Even after destroying it twice and spinning it up and provisioning once after each time it would still fail. This morning I destroyed the box, erased all the files on my Windows 10 machine in that directory, unzipped the files again, made the exact same modifications to config.yml and Vagrantfile, ran vagrant up, and it worked.
I tried the same experiment yesterday with another box and it was able to succeed after 4 provisions, each time the failure was due to Ansible failing to install, and not related to Ubuntu or PHP. I did not have to destroy that box. 
My strong belief now is that the Drupal 8 VM setup process DOES work, but it works approximately 20% of the time, or less -- when on a Windows 10 machine.
My advice to you is this... if your failure is related to Ansible, run `vagrant provision` until the box spins up... even if you have to do that 10 times, keep doing it until it works.
If your failure is not related to Ansible (i.e. ANYTHING else) destroy the box, and possibly delete all the files and unzip the files again. You may keep your IP address and hostname in your hosts file the same, just remember to change your default.config.yml and Vagrantfile to desired settings for VM size, hostname, IP, etc. 

> TL;DR: If you run into ansible failures, provision the box 100 times until it works. If you run into errors other than Ansible, nuke the thing from orbit and unzip the files from scratch until you get a working box (or see Ansible fail, at which time provision the box until it works).
